.nvmrc

```diff
@@ -0,0 +1 @@
+12.22
```

Comment:
Are you sure we can't use anything more modern here? The latest LTS version is 16.x.....

Reply:
Nope, the actions run in `node12`, I tried.

Reply:
Though this only applies for local environments, and does not change anything about the action environment itself, which is controlled in the `action.yml` file.

However, again, making it so that only people with node12 (or want to switch to it) seems rather heavy handed to ensure node12 compliance...